indico/modules/rb/__init__.py

```diff
@@ -42,8 +42,13 @@
     'notification_before_days_weekly': 5,
     'notification_before_days_monthly': 7,
     'notifications_enabled': True,
+    'notification_before_end_daily': 1,
```

Comment:
not sure about the defaults, maybe 7 for everything makes more sense?

Reply:
Yeah, those are the initial values that I came up with. And I think it is something we should discuss.